Content.Server/Chemistry/Components/ReagentDispenserComponent.cs

```diff
@@ -47,6 +47,10 @@ public class ReagentDispenserComponent : SharedReagentDispenserComponent, IActiv
 
         [ViewVariables] [DataField("pack")] private string _packPrototypeId = "";
 
+        [ViewVariables] [DataField("emagPack")] public string EmagPackPrototypeId = "";
```

Comment:
add a prototypeidserializer here for the pack

and while ur in there, add one for the regular packid too